packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -2773,4 +2774,35 @@ export default class Meeting extends StatelessWebexPlugin {
 
     return reject('Meeting:index#sendDTMF --> cannot send DTMF, meeting does not have a connection to the "locus" call control service. Have you joined?');
   }
+
+  /**
+   * Sets the quality of the local video stream
+   * @param {String} level {LOW|MEDIUM|HIGH}
+   * @returns {Promise}
+   */
+  setLocalVideoQuality(level) {
```

Comment:
Might need to rethink about this function as its doing nothing other then getUserMedia with lower resolution then doing updateMedia 

Do we still need to have different function to downgrade incoming video and keep it independent of this

Reply:
The scope of this PR is limited to just the resolution of the sending media. We will have another for the receive media resolution.